seacrowd/sea_datasets/etos/etos.py

```diff
@@ -0,0 +1,206 @@
+                    "deprel": datasets.Sequence(datasets.Value("string")),
+                    "deps": datasets.Sequence(datasets.Value("string")),
+                    "misc": datasets.Sequence(datasets.Value("string")),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_seq_label":
+            features = schemas.seq_label_features(self.UPOS_TAGS)
+
+        else:
+            raise NotImplementedError(f"Schema '{self.config.schema}' is not defined.")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """
+        Returns SplitGenerators.
+        """
+
+        train_path = dl_manager.download_and_extract(_URLS)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": train_path,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """
+        Yields examples as (key, example) tuples.
+        """
+
+        idx = 0
+        with open(filepath, "r", encoding="utf-8") as data_file:
+            tokenlist = list(conllu.parse_incr(data_file))
+            for sent in tokenlist:
+                if "sent_id" in sent.metadata:
+                    sent_id = sent.metadata["sent_id"]
+                else:
+                    sent_id = idx
+
+                tokens = [token["form"] for token in sent]
+
+                if "text" in sent.metadata:
+                    txt = sent.metadata["text"]
+                else:
+                    txt = " ".join(tokens)
+
+                if self.config.schema == "source":
+                    yield idx, {
+                        "sent_id": str(sent_id),
+                        "text": txt,
+                        "tokens": [token["form"] for token in sent],
+                        "lemmas": [token["lemma"] for token in sent],
+                        "upos": [token["upos"] for token in sent],
+                        "xpos": [token["xpos"] for token in sent],
+                        "feats": [str(token["feats"]) for token in sent],
+                        "head": [str(token["head"]) for token in sent],
+                        "deprel": [str(token["deprel"]) for token in sent],
+                        "deps": [str(token["deps"]) for token in sent],
+                        "misc": [str(token["misc"]) for token in sent],
+                    }
+
+                elif self.config.schema == "seacrowd_seq_label":
+                    yield idx, {
+                        "id": str(sent_id),
+                        "tokens": [token["form"] for token in sent],
```

Comment:
```suggestion
                        "tokens": tokens,
```
Similar to previous.